app/main.py

```diff
@@ -1,13 +1,18 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {"errors": [format_linter_error(i) for i in errors], "path": file_path, "status": "passed" if len(errors) == 0 else "failed"}
```

Comment:
Bad code style here - line is too long:
app/main.py:12:80: E501 line too long (136 > 79 characters)